api/client/commands.go

```diff
@@ -1929,6 +1929,10 @@ func (cli *DockerCli) CmdLogs(args ...string) error {
 		return err
 	}
 
+	if env.GetSubEnv("HostConfig").GetSubEnv("LogConfig").Get("Type") != "json-file" {
```

Comment:
It looks like this is returning an nil reference on windows. Maybe it would be safer to check the returned values instead of chaining them all together:

https://jenkins.dockerproject.com/job/Windows-PRs/307/console